lib/module.js

```diff
@@ -21,6 +21,8 @@ module.exports = function sentry (moduleOptions) {
     publishRelease: process.env.SENTRY_PUBLISH_RELEASE || false,
     disableServerRelease: process.env.SENTRY_DISABLE_SERVER_RELEASE || false,
     disableClientRelease: process.env.SENTRY_DISABLE_CLIENT_RELEASE || false,
+    attachCommits: process.env.SENTRY_AUTO_ATTACH_COMMITS !== '0',
```

Comment:
why zero? 
Better to be same as other option e.g.
process.env.SENTRY_AUTO_ATTACH_COMMITS || false